keyboards/senselessclay/had60/info.json

```diff
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "had60",
+    "url": "https://senselessclay.com/",
+    "maintainer": "hadi",
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+      "LAYOUT_60_all": {
```

Comment:
This name does not exist within `had60.h`

Reply:
Addressed!